app/components/dashboard-item/component.ts

```diff
@@ -0,0 +1,36 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { alias, not } from '@ember/object/computed';
+import moment from 'moment';
+
+export default class DashboardItem extends Component {
+    private = not('node.root.id');
+
+    ancestry = computed('node', 'node.{parent,root}', function(): string {
+        const rootId = this.get('node.root.id');
+        const rootLink = this.get('node.root.links.self');
+        const parentId = this.get('node.parent.id');
+        const grandpaLink = this.get('node.parent.links.relationships.parent.links.related.href');
+
+        if (!rootId || !parentId) {
+            return parentId ? this.get('node.parent.title') : '';
+        }
+
+        const items = [this.get('node.root.title')];
+        const separator = ' / ';
+
+        if (grandpaLink === rootLink) {
+            items.push(this.get('node.parent.title'));
+        } else if (parentId !== rootId) {
+            items.push('\u2026', this.get('node.parent.title'));
```

Comment:
Is there a more readable way to refer to ellipses, or is it worth the cognitive load to memorize (i.e. regularly google) what `u2026` means?

Reply:
`&hellip;` works in HTML, but in JS, `\u2026` is the only way to do it AFAICT. This is also something that will need to be refactored later for i18n (RTL, the character representation, and possibly even structurally with the conventions of the language). We can put this in the translations file for now though.

https://en.wikipedia.org/wiki/Ellipsis#Computer_representations